src/amo/reducers/addonsByAuthors.js

```diff
@@ -181,16 +185,21 @@ const reducer = (
       }
 
       newState.loadingFor[joinAuthorNamesAndAddonType(
-        action.payload.authorUsernames, action.payload.addonType)] = true;
+        action.payload.authorUsernames, addonType)] = true;
+
       newState.byAuthorNamesAndAddonType[joinAuthorNamesAndAddonType(
-        action.payload.authorUsernames, action.payload.addonType)] = null;
+        action.payload.authorUsernames, addonType)] = null;
 
       return newState;
     }
     case LOAD_ADDONS_BY_AUTHORS: {
       const newState = deepcopy(state);
-      const pageSize = action.payload.addonType === ADDON_TYPE_THEME ?
-        THEMES_BY_AUTHORS_PAGE_SIZE : EXTENSIONS_BY_AUTHORS_PAGE_SIZE;
+      const { addonType: type } = action.payload;
+
+      const addonType = getAddonTypeFilter(type);
+      const pageSize = ADDON_TYPE_THEMES.includes(type) ||
+        addonType === ADDON_TYPE_THEMES_FILTER ? THEMES_BY_AUTHORS_PAGE_SIZE :
```

Comment:
Why do you want to check `ADDON_TYPE_THEMES_FILTER`?

Reply:
I was using the modified type in saga for this reducer so addonType came through as the modified type but it seems we can remove this here too